R/source.R

```diff
@@ -5,13 +5,18 @@
 #' @param path Path to files.
 #' @param pattern Regular expression used to filter files.
 #' @param env Environment in which to evaluate code.
+#' @param label If not-`NULL`, will run only test with this label.
 #' @param chdir Change working directory to `dirname(path)`?
 #' @param wrap Automatically wrap all code within [test_that()]? This ensures
 #'   that all expectations are reported, even if outside a test block.
 #' @export
 #' @keywords internal
-source_file <- function(path, env = test_env(), chdir = TRUE,
-                        wrap = TRUE) {
+source_file <- function(path,
+                        env = test_env(),
+                        chdir = TRUE,
+                        label = NULL,
```

Comment:
Do you think it's worth leaving the door open to multiple labels here?

Reply:
I'd rather start with a single label, and then later expand, if needed.